x-pack/plugins/security/server/authentication/api_keys/api_keys.ts

```diff
@@ -335,6 +337,32 @@ export class APIKeys {
     return result;
   }
 
+  /**
+   * Tries to validate an API key.
+   * @param apiKey ApiKey.
+   */
+  async validate(apiKey: Pick<CreateAPIKeyResult, 'id' | 'api_key'>) {
+    if (!this.license.isEnabled()) {
+      return { isValid: false };
+    }
+
+    const fakeRequest = getFakeKibanaRequest(apiKey);
+
+    this.logger.debug(`Trying to validate an API key as current user`);
```

Comment:
nit: "as current user" doesn't apply in this context. We don't know the owner of this API Key, so we can't say if it belongs to the "current user" or not.

```suggestion
    this.logger.debug(`Trying to validate an API key`);
```